frontend/app/components/api/api-v3/hal-resources/work-package-resource.service.ts

```diff
@@ -69,27 +69,27 @@ function wpResource(HalResource:typeof op.HalResource, NotificationsService:any,
       this.getForm()
         .catch(deferred.reject)
         .then(form => {
-          var plain_payload = form.payload.$source;
+          var plainPayload = form.payload.$plain();
```

Comment:
This change breaks changing assignees. I verified when using $source again, the functionality was restored. What exactly is the difference between the two?